launchable/test_runners/maven.py

```diff
@@ -1,22 +1,47 @@
 import os
+from typing import List
+from xmlrpc.client import Boolean
 import click
 from . import launchable
 
 
-@click.argument('source_roots', required=True, nargs=-1)
+@click.option('--from-file', 'from_files', required=False, multiple=True, type=click.Path(exists=True))
+@click.argument('source_roots', required=False, nargs=-1)
 @launchable.subset
-def subset(client, source_roots):
+def subset(client, source_roots, from_files):
+
+    def is_file(f: str) -> Boolean:
```

Comment:
[question]
do we have to have this return in `xmlrpc.client.Boolean` instead of Python's original `bool` type?

Reply:
Oh, sorry. It's my miss. I'll fix it. Thank you!

Reply:
done https://github.com/launchableinc/cli/pull/410/commits/dc42cacddbf75837bb8aa872a1f0a5f87c152c5f